controllers/comments.js

```diff
@@ -0,0 +1,21 @@
+const { removeCommentById } = require("../models/comments");
+
+module.exports.deleteCommentById = (request, response, next) => {
+	const { comment_id } = request.params;
+	removeCommentById(comment_id)
+		.then((result) => {
+			if (result.rowCount === 1) {
+				response.status(204).send({
+					message: `Comment with ${comment_id} succesfully deleted.`,
```

Comment:
The ticket asked that you return no content when you've deleted a comment.
I'd stick with what the ticket asks for as a message like this is most likely something you'd display in the front end not the backend.

Reply:
Ok I see, will remove, thanks !